build.psm1

```diff
@@ -1097,6 +1097,9 @@ Restore the module to '$Pester' by running:
             $script:nonewline = $true
             $script:inerror = $false
         }
+        elseif ($trimmedline -match "^\d+ms$") {
```

Comment:
Some tests go into seconds. This regex will not work for those.

Describing long
  [+] test 2.04s

Reply:
You can use "^\d+m?s$" unless minutes is possible then use something like "\d+(m?s|m)$"

Reply:
Pester does not go to minutes 

Describing long
  [+] test 62.03s

Reply:
Good catch @adityapatwardhan. I updated the pattern to be `"^\d+(\.\d+)?m?s$"`.
More information is added to the PR description.